model/client4.go

```diff
@@ -116,6 +116,7 @@ func (c *Client4) Must(result interface{}, resp *Response) interface{} {
 }
 
 func NewAPIv4Client(url string) *Client4 {
+	url = strings.TrimSuffix(url, "/")
```

Comment:
Would we consider using `strings.TrimRight`, to get the automatic removal of multiple slashes? Easy to unit test as well.

Reply:
Great point. Changed to `TrimRight()`.  Didn't realize that method was hanging out there!